Makefile

```diff
@@ -74,6 +74,7 @@ verify-proto:
 .PHONY: apidocs
 apidocs:
 	./hack/api-docs/generate.sh docs-src/spec.md
+	$(MAKE) docs
```

Comment:
I'd assumed that `docs` and `apidocs` were separate targets. Suggest we don't add `apidocs` here, but rather add `docs` to the `generate` target so that everything gets build from there?

Reply:
Or we remove the `apidocs` target and just have a single `docs` ...

Reply:
+1 fewer makefile targets the better...

Reply:
Good call, I've added `docs` to `generate` and the API docs generation to `docs`.